doc/source/examples/unresolved-cfd-dem/gas-solid-spouted-cylinder-bed/gas-solid-spouted-cylinder-bed.rst

```diff
@@ -0,0 +1,312 @@
+
+.. image:: images/geometry.png
+    :alt: The geometry and boundary conditions
+    :align: center
+    :name: geometry
+    :height: 10cm 
+
+The geometry of the bed was created using `Pointiwise <../../../tools/pointwise/pointowise.html>`_, and the overview of created mesh is also shown below;
+
+.. image:: images/mesh.png
+    :alt: The geometry and boundary conditions
+    :align: center
+    :name: mesh_ver
+    :height: 10cm
+
+In Unresolved-CFD-DEM, Cells need to have a enough volume to contain particles in it. Basically the size of cells should be least three time larger than the diameter of the particles in order to get stable calculation. So this can be expressed as follows.
+
+.. math:: 
+  \dfrac{d_p}{\Delta x} \leq 3
+
+where :math:`d_p` is the diameter of the particles, and :math:`\Delta x` is the characteristic size of the cell. In this example, we are dealing with the particles 5mm in diameter, which means we need at least 15mm for the size of cells. Also, we need to apply relatively small meshes to the short channel below. So We set the size of the central cylinder to 15mm, and made other grid coarser.
+
+
+Lagrangian Physical Properties
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The physical properties section is almost the same as the another example. Here, parameter for the direction of gravity, the diameter and density of particles, and the number of particles are modified. We insert 100,000 particles with a 5mm diameter in this simulation. 
+
+.. code-block:: text
+
+    subsection lagrangian physical properties
+      set gx                       = 0
+      set gy                       = 0
+      set gz                       = -9.81
+      set number of particle types = 1
+      subsection particle type 0
+        set size distribution type            = uniform
+        set diameter                          = 0.0025
+        set number                            = 100000
+        set density particles                 = 500
+        set young modulus particles           = 10000000
+        set poisson ratio particles           = 0.25
+        set restitution coefficient particles = 0.97
+        set friction coefficient particles    = 0.4
+        set rolling friction particles        = 0.3
+      end
+      set young modulus wall           = 10000000
+      set poisson ratio wall           = 0.25
+      set restitution coefficient wall = 0.33
+      set friction coefficient wall    = 0.2
+      set rolling friction wall        = 0.3
+    end
+
+Insertion Info
+~~~~~~~~~~~~~~~~~~~
+
+The ``insertion info`` subsection manages the insertion of particles. It should be modified because we changed the number of particles, and use different geometry. The parameter of insertion box was set as follows so that it could fit in the cylinder.
```

Comment:
```suggestion
The ``insertion info`` subsection manages the insertion of particles. This section has to be modified because of the change in particle number and use of a different geometry. The insertion box parameter was set as follows so that it could fit in the cylinder.
```

Reply:
"and the use of a different geometry"

Reply:
"as follows, so that"